apps/web/components/ReviewsAccordion/ReviewsAccordion.vue

```diff
@@ -9,14 +9,48 @@
           {{ t('customerReviews') }}
         </h2>
       </template>
-      <SfButton
-        @click="isAuthorized ? openReviewModal() : openAuthentication()"
-        data-testid="create-review"
-        class="mt-2 mb-4"
-        size="base"
-      >
-        {{ t('createCustomerReview') }}
-      </SfButton>
+
+      <div>
+        <div class="flex my-2">
+          <div class="w-1/2 flex flex-col">
+            <p class="text-center text-sm">Average Rating</p>
+            <div class="flex justify-center">
+              <SfRating
+                class="pb-2"
+                size="lg"
+                :max="5"
+                :value="reviewGetters.getAverageRating(productReviewAverage)"
+                :half-increment="true"
+              />
+              <h3 class="font-bold text-xl ml-2 flex">{{ reviewGetters.getAverageRating(productReviewAverage) }}</h3>
+            </div>
+            <p class="text-xs text-center text-">based on {{ totalReviews }}Review(s)</p>
+            <SfButton
+              @click="isAuthorized ? openReviewModal() : openAuthentication()"
+              data-testid="create-review"
+              class="mt-2 mb-4 w-1/2 mx-auto"
+              size="base"
+            >
+              {{ t('createCustomerReview') }}
+            </SfButton>
+          </div>
+
+          <div class="flex flex-col">
+            <div v-for="(splitReview, key) in ratingPercentages" :key="key" class="flex items-center">
+              <p class="w-4 text-center">{{ 6 - key }}</p>
+              <SfRating class="mx-2 pb-1" size="base" :max="1" :value="1" />
+              <SfProgressLinear
+                class="self-center"
+                size="sm"
+                :value="splitReview"
+                aria-label="5star-Rating-Progress-Element"
```

Comment:
Does this really describe what the element shows?

Reply:
Not really, i already have it changed in an uncommitted change. Will push it soon, after i improved some parts of the new written code. @Tim-M-S